nodejs/week3/Mealsharing/src/backend/api/reservations.js

```diff
@@ -0,0 +1,94 @@
+const knex = require("../database");
+const express = require("express");
+const router = express.Router();
```

Comment:
Similar to the previous file consider validating the inputs and responding with accurate status codes.